addons/website/views/snippets/s_key_benefits.xml

```diff
@@ -4,32 +4,30 @@
 <template id="s_key_benefits" name="Key benefits">
     <section class="s_key_benefits pt48 pb48">
         <div class="container">
-            <div class="row o_grid_mode s_nb_column_fixed" data-row-count="11" style="column-gap: 48px;">
-                <div class="o_grid_item g-col-lg-4 g-height-1 col-lg-4" style="--grid-item-padding-x: 16px; --grid-item-padding-y: 0px; grid-area: 1 / 1 / 2 / 5; z-index: 1;">
```

Comment:
@chgo-odoo Apparently, you need to adapt `design-themes` here (e.g., the `xpath` [here](https://github.com/odoo/design-themes/blob/master/theme_artists/views/snippets/s_key_benefits.xml#L22) will fail since you only have 3 `.col-lg-4` now) :grimacing:.